indico/modules/events/persons/placeholders.py

```diff
@@ -63,6 +64,27 @@ def render(cls, person, event, **kwargs):
                                                                           title=event.title)
 
 
+class EventContributionsPlaceholder(ParametrizedPlaceholder):
+    name = 'event_contributions'
+    description = _("Event contributions, optional: One where speaker")
+    param_friendly_name = 'onlyspeakers'
+    param_required = False
+
+    @classmethod
+    def render(cls, param, person, event, **kwargs):
+        res = "<ul>\n"
+        for c in get_contributions_for_person(event, person, (param and param == 'onlyspeakers')):
```

Comment:
`param and` isn't needed, comparing an empty string or None against another string will already be false